addons/mrp/models/mrp_production.py

```diff
@@ -2794,8 +2801,16 @@ def _get_product_catalog_order_data(self, products, **kwargs):
         product_catalog = super()._get_product_catalog_order_data(products, **kwargs)
         for product in products:
             product_catalog[product.id] |= self._get_product_price_and_data(product)
+            product_catalog[product.id] |= self._get_product_quantity(product)
         return product_catalog
 
+    def _get_product_quantity(self, product):
+        self.ensure_one()
+        move = self.move_raw_ids.filtered(lambda x: x.product_id.id == product.id)
+        if not move:
+            return {}
+        return {'quantity': move.product_uom_qty}
+
```

Comment:
what is this for?

Reply:
same as the previous.

Reply:
Same issue as the `MrpBomLine._get_product_catalog_lines_data()`except this one was implemented in `stock.stock_move`.